client/src/features/calendar/DayCard.js

```diff
@@ -4,13 +4,18 @@ import Event from "./Event";
 const DayCard = ({ day, month, events }) => {
   // Styling for bullet point
 
+    // Determine the current day
+    const currentDay = new Date().getDate();
+    //Aplies styling if true
+    const currentDayStyles = day === currentDay ? { backgroundColor: "#BFD0D8" } : {};
```

Comment:
won't this style on every month where the day of the month is the current day? e.g. today (12/23) would also highlight 1/23/2023, 2/23/2023, etc

Reply:
Um, yeah my bad. Let me fix that.

Reply:
[date-fns `isSameDay`](https://date-fns.org/v2.29.3/docs/isSameDay) might be helpful here 🙂

Reply:
> [date-fns `isSameDay`](https://date-fns.org/v2.29.3/docs/isSameDay) might be helpful here 🙂

How can I use isSameDay if DayCard only has day, month?

Reply:
@timmyichen I implemented isSameDay as you suggested. Is there a better implement the conversion of month into an integer for the isSameDay check?

```
const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
    const monthInt = months.indexOf(month);
```

Reply:
Ah I didn't realize individual pieces of the date were being passed down through props... I think ideally, you just construct a Date object (from the day + month + year) and pass that through - that way you can just pass this date + the startDay of the event.

Otherwise, you can use the [date-fns `parse`](https://date-fns.org/v2.29.3/docs/parse) function (specifying that the input format contains a fully-spelled out month)

Alternatively, you can construct a date using "MonthSpelledOut Day, Year" (no need to get indexes):

<img width="425" alt="image" src="https://user-images.githubusercontent.com/5304277/209423349-13aaa878-d000-4e30-9748-14dcb3b8790b.png">

Reply:
> Ah I didn't realize individual pieces of the date were being passed down through props... I think ideally, you just construct a Date object (from the day + month + year) and pass that through - that way you can just pass this date + the startDay of the event.
> 
> Otherwise, you can use the [date-fns `parse`](https://date-fns.org/v2.29.3/docs/parse) function (specifying that the input format contains a fully-spelled out month)
> 
> Alternatively, you can construct a date using "MonthSpelledOut Day, Year" (no need to get indexes):
> 
> <img alt="image" width="425" src="https://user-images.githubusercontent.com/5304277/209423349-13aaa878-d000-4e30-9748-14dcb3b8790b.png">

I tried both the [parse function with the props](https://github.com/Caleb-Cohen/Together/pull/200/commits/c5893e1f3d19aa55cbbebac70f7bb8e9837e7c49) and creating an [object  before passing it to dayCard](https://github.com/Caleb-Cohen/Together/pull/200/commits/cccf6dac73fd46e65102f329b08d4590a049b2d7). The latter was a bit tougher, but I like it more as well.

Any other feedback before I merge this?

Reply:
No feedback other than it looks great!  Thanks :)